src/QsCompiler/BondSchemas/Protocols.cs

```diff
@@ -147,7 +147,13 @@ private static SimpleBinarySerializer GetSimpleBinarySerializer()
         private static Task<SimpleBinaryDeserializer> QueueSimpleBinaryDeserializerInitialization()
         {
             VerifyLockAcquired(BondSharedDataStructuresLock);
-            return Task.Run(() => new SimpleBinaryDeserializer(typeof(QsCompilation)));
+
+            // The inlineNested constructor argument is set to false because it optimizes deserialization time at the expense of initialization time and memory.
```

Comment:
To clarify, is this comment ("optimizes deserialization time at the expense of initialization time and memory") describing what happens when `inlineNested` is true? To me, the phrasing sounds like it is saying that we are setting `inlineNested` to false in order to optimize deserialization time at the expense of initialization time.

Reply:
Yes, I'll change the comment to make it clear.

Reply:
Change the comment to:
`The inlineNested constructor argument is set to false because when it is set to true deserialization time is optimized at the expense of initialization time and memory.`

Hopefully now it is clearer.

Reply:
Yes, it looks correct now, thanks.

Maybe it's just me, but IMO it's still a little hard to determine the purpose of the comment if I was reading it without context. The key question is "why is inlineNested set to false?" but the comment only answers that indirectly. Maybe something simpler and more direct like this would help?

> inlineNested is false in order to decrease the time needed to initialize the deserializer. While this setting may also increase deserialization time, we did not notice any performance drawbacks with our Bond schemas.

Reply:
I liked it! The comment is now updated.